tests/functional/common.config

```diff
@@ -72,6 +72,11 @@ export SED_PROG="`set_prog_path sed`"
 export BC_PROG="`set_prog_path bc`"
 [ "$BC_PROG" = "" ] && _fatal "bc not found"
 
+PGREP_OPTIONS='-a'
+if pgrep * -a 2>&1 | grep -q "invalid option";then
```

Comment:
Sorry, I misunderstood the problem. You need to change line 75 from
PGREP_OPTIONS='-a'
to
export PGREP_OPTIONS='-a'

it works for my env